compiler/src/iree/compiler/Codegen/LLVMCPU/DispatchABI.cpp

```diff
@@ -490,7 +490,7 @@ HALDispatchABI::getInputTypes(MLIRContext *context,
 // static
 LLVM::DISubprogramAttr
 HALDispatchABI::buildScopeAttr(mlir::ModuleOp moduleOp, StringRef funcName,
-                               LLVMTypeConverter *typeConverter) {
+                               const LLVMTypeConverter *typeConverter) {
```

Comment:
Uggh! `const` everywhere...  (Not issue with the PR, I understand this is forced due to upstream change)